### DISCUSSION TITLE
[Homepage] Add scrollbar

### DISCUSSION THREAD

**Discussion by F-park** [laike9m/Python-Type-Challenges#112] [General]
There are too many advance challenges in homepage, we cannot see random button directly.

![image](https://github.com/laike9m/Python-Type-Challenges/assets/52167622/9360f17a-8239-4948-9595-076256d4f2ba)

![image](https://github.com/laike9m/Python-Type-Challenges/assets/52167622/eefcb30a-7aa3-4326-867f-10837d4dc48e)

**Comment by laike9m** [comment]
> we cannot see random button directly.

Honestly, I don't see this as a problem
